Fatras/include/ActsFatras/Digitization/detail/ParametersCast.hpp

```diff
@@ -0,0 +1,75 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2016-2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Utilities/Definitions.hpp"
+#include "Acts/Utilities/Helpers.hpp"
+#include "Acts/Utilities/ParameterDefinitions.hpp"
+
+namespace ActsFatras {
+namespace detail {
+
+/// Cast function definition for bound parameters
+using BoundCastFunction =
+    std::function<double(const Acts::Vector2D&, const Acts::Vector3D&, double)>;
```

Comment:
Nitpicking: define this near the bound casts array/ function. This is only needed there.